internal/controllers/common/resource_definitions/resource_definitions.go

```diff
@@ -550,7 +587,10 @@ func NewPodForReports(cr *operatorv1beta1.Cryostat, imageTags *ImageTags, tls *T
 				SecurityContext: containerSc,
 			},
 		},
-		Volumes:         volumes,
+		Volumes:              volumes,
+		NodeSelector: nodeSelector,
+		Affinity:     affinity,
+		Tolerations:  tolerations,
```

Comment:
`go fmt` wants these assignments to be aligned.
```suggestion
		Volumes:         volumes,
		NodeSelector:    nodeSelector,
		Affinity:        affinity,
		Tolerations:     tolerations,
```